auth-api/src/auth_api/resources/org.py

```diff
@@ -146,6 +146,7 @@ def put(org_id):
         valid_format, errors = schema_utils.validate(request_json, 'org')
         toke_info = g.jwt_oidc_token_info
         bearer_token = request.headers['Authorization'].replace('Bearer ', '')
+        origin = request.environ.get('HTTP_ORIGIN', 'localhost')
```

Comment:
what is the use of origin URL ?

Reply:
We use the origin_url to send notification mail. Since, am yet to get nod on whether to send mail to staff ( similar to bceid), so removing it